public/js/cohortStudent.js

```diff
@@ -62,3 +63,34 @@ addStudentButton.addEventListener('click', (e) => {
     swal('Please Enter Data ! ! ', '', 'error');
   }
 });
+
+deleteStudentButton.forEach((element) => {
```

Comment:
@lubnaabd Had made a function like this in `delCohort.js`, see the similarities and put them in one function, then call them back here @HemaSAli .

Reply:
I saw what lubna made , I think they are the same 
and here is lubna code 
```
Array.prototype.forEach.call(del, (element) => {
  element.addEventListener('click', (event) => {
```

Reply:
I know they're the same, don't have repeated code. Pick Lubna's code and assign it to a function, then use it in your code and Lubna's.